core/src/main/scala/org/apache/spark/ui/env/EnvironmentPage.scala

```diff
@@ -80,9 +81,10 @@ private[ui] class EnvironmentPage(
             <a>Hadoop Properties</a>
           </h4>
         </span>
-        <div class="aggregated-hadoopProperties collapsible-table collapsed">
-          {hadoopPropertiesTable}
-        </div>
+          <div class="aggregated-hadoopProperties collapsible-table collapsed">
+            {hadoopPropertiesTable}
```

Comment:
I think this indent change was unintentional?
I'm not sure this change is needed. Now, it is clear that there are no properties available, rather than just making the section go missing.

Reply:
@srowen 
Thanks for review.
Handled the indentation issue.